missions/a8/transcripts/_meta

```diff
@@ -1,82 +1,406 @@
 {
     "name": "a8",
-    "featured": false,
-    "incomplete": true,
+    "featured": true,
+    "incomplete": false,
```

Comment:
I think in future we should separate this stuff out from getting the missions done, since we can then deploy as a "hidden" mission and check there, which allows for more people to do a final pass than if they have to check out a branch and rebuild locally.